ms2deepscore/models/EmbeddingEvaluatorModel.py

```diff
@@ -68,46 +70,41 @@ def save(self, filepath):
         }
         torch.save(settings_dict, filepath)
 
-    def train_evaluator(
-            self,
-            data_generator: DataGeneratorEmbeddingEvaluation,
-            mini_batch_size: int,
-            batches_per_iteration: int,
-            learning_rate: float,
-            num_epochs: int,
-            val_generator: DataGeneratorEmbeddingEvaluation = None):
+    def train_evaluator(self,
```

Comment:
```suggestion
    def train(self,
```

Reply:
Since it is now part of the model class we don't have to specify what this is training.

Reply:
There is a method of the nn.module which is also called train, which we call in train_evaluator. Which I think would result in us breaking something, but I am not fully sure.

Reply:
Oh sorry, that's of course correct. Than nevermind....